sdk/communication/Azure.Communication.Common/tests/CommunicationTestEnvironment.cs

```diff
@@ -14,19 +14,23 @@ public class CommunicationTestEnvironment : TestEnvironment
         private const string SkipIntSmsTestEnvironmentVariableName = "SKIP_INT_SMS_TEST";
         private const string SkipIntPhoneNumbersTestEnvironmentVariableName = "SKIP_INT_PHONENUMBERS_TEST";
 
-        public string LiveTestDynamicConnectionString => GetRecordedVariable(LiveTestDynamicConnectionStringEnvironmentVariableName);
+        public string LiveTestDynamicConnectionString => GetRecordedVariable(
+            LiveTestDynamicConnectionStringEnvironmentVariableName,
+            options => options.HasSecretConnectionStringParameter("accessKey", SanitizedValue.Base64));
 
         public Uri LiveTestDynamicEndpoint => new Uri(Core.ConnectionString.Parse(LiveTestDynamicConnectionString).GetRequired("endpoint"));
 
         public string LiveTestDynamicAccessKey => Core.ConnectionString.Parse(LiveTestDynamicConnectionString).GetRequired("accesskey");
 
-        public string LiveTestStaticConnectionString => GetRecordedVariable(LiveTestStaticConnectionStringEnvironmentVariableName);
+        public string LiveTestStaticConnectionString => GetRecordedVariable(
+                LiveTestStaticConnectionStringEnvironmentVariableName,
+                options => options.HasSecretConnectionStringParameter("accessKey", SanitizedValue.Base64));
 
         public Uri LiveTestStaticEndpoint => new Uri(Core.ConnectionString.Parse(LiveTestStaticConnectionString).GetRequired("endpoint"));
 
         public string LiveTestStaticAccessKey => Core.ConnectionString.Parse(LiveTestStaticConnectionString).GetRequired("accesskey");
 
-        public string CommunicationTestPhoneNumber => GetVariable(AzurePhoneNumber);
+        public string CommunicationTestPhoneNumber => GetRecordedVariable(AzurePhoneNumber, options => options.IsSecret("+14255550123"));
```

Comment:
Not all tests were using this as a variable in recordings, but I think this change is okay because for the tests that do not expect this to be in the recording, the value is hard-coded in playback.